lib/TierCortex.ts

```
@@ -1,265 +1,294 @@
+        const encodedState: EncodedState = {
+            tiers: simplifiedTiers
+        };
+        if (title) {
+            encodedState.title = title;
+        }
+        const jsonString = JSON.stringify(encodedState);
+        return LZString.compressToEncodedURIComponent(jsonString);
+    }
+
+    public isCustomItem(itemId: string): boolean {
+        return !this.packageItemLookup.hasOwnProperty(itemId);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)